OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -73,6 +73,9 @@
     <StartAction>Project</StartAction>
     <StartArguments>Game.Mod=ts</StartArguments>
   </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
```

Comment:
Note that the executables that we ship for windows are built using mono and the makefile, so that may also need to be updated if this changes compilation flags.